svelte-kit-scss/src/lib/helpers/file.ts

```diff
@@ -86,8 +87,7 @@ export const remapFileContents = (
 ): FileContents | undefined => {
   return responseData
     ? {
-        // content: Buffer.from(responseData.content, 'base64').toString('binary'),
```

Comment:
Would like to request if we can use Buffer.from had changed it with Adam trying to fix adapter but it did not solve the issue. You can revert to BUffer